qiime2/tests/test_artifact_api.py

```diff
@@ -252,6 +252,22 @@ def test_metadata_column_from_factory(self):
     ints=ints,
     metadata=mdc,
 )
+"""
+        self.assertEqual(exp, use.render())
+
+    def test_use_merge_feature_table(self):
+        action = self.plugin.actions['variadic_input_method']
+        use = ArtifactAPIUsage()
+        action.examples['feature_table_merge_example'](use)
+
+        exp = """\
+from qiime2.plugins.dummy_plugin.methods import variadic_input_method
+
+out, = variadic_input_method(
+    ints=int,
```

Comment:
Note to @thermokarst - this driver should template in the factory body to make this more clear.